src/user_config_prompts.ts

```diff
@@ -14,7 +14,7 @@ function prompt(context: vs.ExtensionContext, key: string, prompt: () => Thenabl
 	//context.globalState.update(stateKey, undefined);
 
 	// If we've not prompted the user with this question before...
-	if (context.globalState.get(stateKey, false) !== true) {
+	if (context.globalState.get(stateKey) !== true) {
```

Comment:
Thanks! I kept meaning to come back to this. It seems TS is sticking with this and casting a literal `false` to boolean felt dirty to me - I'm much happier with this :D